features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -469,6 +469,9 @@
             They will not be able to access claims from local userstore. To get claims by mapping users with exact same username from local
             userstore (for non LOCAL scenarios) use mapFederatedUsersToLocal config -->
             <UserType>{{oauth.grant_type.saml_bearer.user_type}}</UserType>
+            {% if oauth.grant_type.saml_bearer.use_user_id_from_claims is defined %}
+            <UseUserIdFromClaims>{{oauth.grant_type.saml_bearer.use_user_id_from_claims}}</UseUserIdFromClaims>
```

Comment:
What would this do? Can't see any usage in documentation also. The name itself is not clear.

Reply:
removed the config